src/main/kotlin/io/bazel/kotlin/builder/toolchain/KotlinToolchain.kt

```diff
@@ -63,6 +64,11 @@ class KotlinToolchain private constructor(
       "src", "main", "kotlin",
       "skip-code-gen.jar").toPath()
 
+    private val JDEPS_GEN_PLUGIN = BazelRunFiles.resolveVerified(
+      "io_bazel_rules_kotlin",
```

Comment:
We can't hard code the repository rule, so use the hack when you merge from master.

We really need to pass in the paths via flags. It's on the list.

Reply:
Done